src/platform/site-wide/announcements/config/index.js

```diff
@@ -1,5 +1,17 @@
+// Relative imports.
+import HomepagRedesignModal from '../components/HomepageRedesignModal';
```

Comment:
Notice that the typo persists here, but things still work. This is because what's being imported here is the _default_ export from `../components/HomepageRedesignModal`. That is the HOC that is exported at the bottom of the file. On import, you can name it whatever you'd like. As noted in the other comment, we should not export the unwrapped component, because that would allow somebody to import it `import { HomepageRedesignModal } from ...` and that would be the functionality without the connection to the Redux store. Providing the option to import that would be bad.

I do think the typo should be fixed here as well.

Reply:
fixed type